docs/docs/exosphere/create-graph.md

```
@@ -0,0 +1,298 @@
+        print("Graph template created successfully!")
+        print(f"Validation status: {result['validation_status']}")
+        return result
+    except Exception as e:
+        print(f"Error creating graph template: {e}")
+        raise
+
+# Run the function
+import asyncio
+asyncio.run(create_flow_template())
+```
+## Input Mapping Patterns
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)